lib/gui/app/index.html

```diff
@@ -11,6 +11,13 @@
   </head>
   <body>
     <header class="section-header" ng-controller="HeaderController as header">
+      <button class="button button-link"
+        ng-if="header.shouldShowWifi()"
+        ng-click="header.openWifiSettings()"
+        tabindex="4">
```

Comment:
There's now two buttons with tabindex of 4? (however I don't know if that's actually a problem or not)

Reply:
Since only one element's there at a time with `ng-if` it's fine.